src/js/common/components/form-elements/ErrorableRadioButtons.jsx

```diff
@@ -147,6 +146,7 @@ ErrorableRadioButtons.propTypes = {
   errorMessage: PropTypes.string,
   label: PropTypes.string.isRequired,
```

Comment:
Should this be expanded to allow objects due to line 120 in `src/js/edu-benefits/components/EducationWizard.jsx`?